L1Trigger/L1THGCalUtilities/python/caloTruthCellsProducer_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+from L1Trigger.L1THGCal.hgcalBackEndLayer1Producer_cfi import C2d_parValues
```

Comment:
It has recently been modified: see https://github.com/PFCal-dev/cmssw/blob/hgc-tpg-devel-CMSSW_10_6_0_pre2/L1Trigger/L1THGCal/python/hgcalBackEndLayer1Producer_cfi.py

Reply:
Fixed.